### DISCUSSION TITLE
bug since 2024.1

### DISCUSSION THREAD

**Discussion by rchiileea** [elax46/custom-brand-icons#561] [General]
@elax46 maybe add to instructions due to icons not showing and icon picker not working since update 2024.1.1 for some people... change a sticky to tell them to make sure they have both

```
/hacsfiles/custom-brand-icons/custom-brand-icons.js?hacstag=366862031202420
```
and

```
/local/community/custom-brand-icons/custom-brand-icons.js
```

in resources.

**Comment by elax46** [comment]
> @elax46 maybe add to instructions due to icons not showing and icon picker not working since update 2024.1.1 for some people... change a sticky to tell them to make sure they have both
> 
> ```
> /hacsfiles/custom-brand-icons/custom-brand-icons.js?hacstag=366862031202420
> ```
> 
> and
> 
> ```
> /local/community/custom-brand-icons/custom-brand-icons.js
> ```
> 
> in resources.

I pinned the answer and added a note in the readme

**Reply by elax46** [reply]
[update](https://github.com/elax46/custom-brand-icons?tab=readme-ov-file#dont-see-the-icon)

**Reply by elax46** [reply]
@rchiileea Do you by any chance know if this problem has been reported to the HomeAssistant team? I tried searching both on HACS and in the HA frontend project but I didn't find any users who reported the problem
